### PR TITLE
Edits to the responsive design of the main hero

### PR DESCRIPTION
**Pull request by robasaad**

### CODE REVIEW

File: src/main/webapp/css/ashley-bs-theme-layout.css [hunk 1]
```diff
@@ -563,7 +563,17 @@ hr {
...
+	.navbar-brand {
+		width: 100%;
+		text-align: center;
+		padding: 16px 35px;
+	}
+	.navbar-brand img {
+	  width: 175px;
+	}
+}
+@media (max-width:399px) {
```
Comment (avaliani) [inline src/main/webapp/css/ashley-bs-theme-layout.css:576]: This is to make the navbar fat on the ipad right?

Comment (robasaad) [inline src/main/webapp/css/ashley-bs-theme-layout.css:576]: No, I did this to center the logo in the "medium" screen size. Before it was off-centered.

File: src/main/webapp/css/main.css [hunk 2]
```diff
@@ -1314,32 +1282,100 @@ p {
   -ms-animation-delay: 0.6s;
   animation-delay: 0.6s;
   text-align: center;
-  margin-top: 20px;
 }
+#intro-banner #hero .photo-credit {
+  text-align: center;
+  color: #fff;
+  z-index: 9999;
```
Comment (avaliani) [inline src/main/webapp/css/main.css]: I worry about excessive z-index. This can cause the alert / dialog to hide behind the caption. Can you just use z-index: 1 / 10. .alert-float is at z-index 100. #hero .container is at 33 (need to look at z-index with position relative vs. absolute).

File: src/main/webapp/css/ashley-bs-theme-layout.css (hunk 1 above)
Comment (avaliani) [inline src/main/webapp/css/ashley-bs-theme-layout.css:576]: Ok. Cool. Thanks.

File: src/main/webapp/css/main.css [hunk 3]
```diff
@@ -1179,50 +1176,40 @@ p {
...
   content: '';
   left: 0;
-  bottom: 0;
+  top: 0;
   width: 100%;
-  height: 420px;
-  background: -webkit-linear-gradient(top, rgba(255, 255, 255, 0) 0%, rgba(0, 0, 0, 0) 1%, rgba(0, 0, 0, 0.05) 26%, rgba(0, 0, 0, 0.55) 71%, black 100%);
-}*/
+  height: 160px;
+  background: -moz-linear-gradient(top,  rgba(29,29,5,0.5) 0%, rgba(0,0,0,0) 78%); /* FF3.6+ */
```
Comment (avaliani) [inline src/main/webapp/css/main.css:1206]: Can you put a comment with the link of the generator you used. In case we want to make edits. Thanks. Something like "/\* Generated by .... */"

File: src/main/webapp/css/main.css (hunk 2 above)
Comment (robasaad) [inline src/main/webapp/css/main.css]: Ok. I changed it to "z-index: 1" and it still works. This z-index is necessary though or else the photo credit link won't work.

File: src/main/webapp/css/main.css (hunk 3 above)
Comment (robasaad) [inline src/main/webapp/css/main.css:1206]: Sure

File: src/main/webapp/css/main.css [hunk 4]
```diff
@@ -1314,32 +1282,100 @@ p {
...
+
+
+/* Desktop */
+@media (min-width: 768px) {
+  #intro-banner #hero {
+    background-position: center 30%;
+    height: 510px;
+  }
+  #intro-banner #hero h1.hero-text {
+    text-shadow: 1px 1px 8px rgba(0, 0, 0, 0.5);
```
Comment (avaliani) [inline src/main/webapp/css/main.css:1311]: So no text shadow when the width is smaller than 767 px? On purpose?

Comment (robasaad) [inline src/main/webapp/css/main.css:1311]: Yep. I originally had it but it looked bad/inconsistent.

File: src/main/webapp/partials/events.html [hunk 5]
```diff
@@ -1,20 +1,14 @@
...
-            </div>
+            <h1 class="hero-text animated fadeInDown">Make earning and sharing <em>karma</em> part of your daily routine.</h1>
             <div class="clear"></div>
             <div class="cta animated fadeInDown">
                 <a href="#!/tour" class="button-outline tour">Take a tour</a>
                 <a href="" class="button-outline login" ng-click="SessionManager.login()" ng-hide="isLoggedIn">Login</a>
             </div>
-            <p class="photo-credit">Photo by <a href="https://www.flickr.com/photos/usnavy/5507019848" target="_blank">United States Navy</a></p>
         </div>
+        <p class="photo-credit">Photo by <a href="https://www.flickr.com/photos/usnavy/5507019848" target="_blank">United States Navy</a></p>
```
Comment (avaliani) [inline src/main/webapp/partials/events.html]: Why did you move the photo credit outside of the hero container? Know that it's absolutely positioned. But can it be absolutely positioned with respect to the container?

File: src/main/webapp/css/main.css (hunk 4 above)
Comment (avaliani) [inline src/main/webapp/css/main.css:1311]: Alright cool. Thanks for the info!

File: src/main/webapp/css/main.css (hunk 3 above)
Comment (avaliani) [inline src/main/webapp/css/main.css:1206]: Great thanks!

File: src/main/webapp/css/main.css (hunk 2 above)
Comment (avaliani) [inline src/main/webapp/css/main.css]: Ok. Great!

File: src/main/webapp/partials/events.html (hunk 5 above)
Comment (robasaad) [inline src/main/webapp/partials/events.html]: Because I used the container as the top blue box that appears on mobile devices. Also I want to position the photo credit so that it's absolute to div#hero and not the container.

If you want to keep the photo credit inside the container, we can do so but that requires creating a new div for the top blue box. I'd rather just keep it the way it is. Thoughts?

Comment (avaliani) [inline src/main/webapp/partials/events.html]: Interesting. A new div I think would be nice. I think it's a bootstrap thing not to put things that don't go outside of the container outside of the container. The container is a semantic notion. Technically the photo-credit is container bound. Is there any reason it can't be absolute to the container? The container has relative position. So you can put absolute position elements in it.

Also, if you put it in the container, you probably won't need a z-index for it. Will you? The container already has a z-index.

So to me creating a div with class name "hero-text-and-cta" would be good.

Thoughts?

Comment (avaliani) [inline src/main/webapp/partials/events.html]: Just realized. the blue box is not container bound right? It can span. Let me think about this a bit more.

Comment (robasaad) [inline src/main/webapp/partials/events.html]: If we do that, then we'd have to specify fixed heights for the container for all 4 screen widths.

Comment (avaliani) [inline src/main/webapp/partials/events.html]: Discussed via hangout. Like our solution to update :before on the hero. Thanks in advance for making the changes!
